gpflow/utilities/utilities.py

```diff
@@ -446,3 +448,15 @@ def _str_tensor_value(value: np.ndarray):
         out = f"{brackets}{out}..."
 
     return out
+
+
+def select_parameters_with_prior(parameters: Sequence[Parameter]) -> Sequence[Parameter]:
+    return [p for p in parameters if p.prior is not None]
+
+
+def select_dict_parameters_with_prior(model: tf.Module) -> Dict[str, Parameter]:
+    return {
+        k: p
+        for k, p in parameter_dict(model).items()
+        if hasattr(p, "prior") and p.prior is not None
+    }
```

Comment:
is it really worth maintaining this utility function? :thinking:

Reply:
I don't think that it is big trouble, but saves some space in notebooks.